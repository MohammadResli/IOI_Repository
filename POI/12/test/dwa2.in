20
12 7 20 18 11 19 15 5 12 17 10 13 16
16 10 16 15 13 9 14 12 11 7 4 3 5 6 17 19 18
14 15 9 17 2 5 12 14 7 16 4 18 8 6 10
15 7 11 19 5 15 16 14 6 8 18 17 12 3 2 20
13 15 4 2 14 10 3 9 6 13 17 1 11 12
13 10 5 18 13 16 4 17 15 2 12 3 9 8
13 16 20 2 14 19 10 4 3 1 11 13 8 17
13 19 4 18 17 16 3 12 7 9 13 20 6 15
11 17 2 11 3 6 19 5 8 16 15 14
15 3 12 2 20 19 5 15 7 1 11 17 6 18 14 16
15 10 18 9 15 5 1 7 12 17 14 4 20 13 16 2
15 18 1 3 6 16 13 8 5 10 15 14 17 4 11 2
14 1 12 7 16 15 17 14 5 19 11 20 2 8 6
15 20 7 9 17 19 13 2 5 10 4 3 11 15 16 12
17 5 1 20 3 18 11 8 17 10 13 4 14 12 2 6 19 9
17 14 20 18 1 9 11 10 19 7 3 8 2 12 4 17 6 13
18 7 5 13 10 15 9 2 16 19 3 12 8 20 11 6 4 1 14
13 4 16 6 12 10 1 8 19 15 20 3 2 11
13 18 16 2 10 8 4 17 14 13 7 9 1 15
12 11 15 13 4 1 18 16 7 8 10 14 17
