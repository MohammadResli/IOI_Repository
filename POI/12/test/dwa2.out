8
1 7 10 13 14 15 19 20 
